docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -270,11 +274,7 @@ class ComponentExample extends React.Component<ComponentExampleProps, ComponentE
       rtl: showRtl,
     }
 
-    return (
-      <Provider theme={newTheme}>
-        <SourceRender.Consumer>{({ element }) => element}</SourceRender.Consumer>
-      </Provider>
-    )
+    return <Provider theme={newTheme}>{element}</Provider>
```

Comment:
Nope, it will called with the result of rendering i.e. an element from `ButtonExample`. This result will be will wrapped with our `Provider` and then will be passed to `renderExampleFromCode()`. It's quite similar to `render*` props that we had before.

Reply:
Got it, thanks for the clarification.